src/pages/TestRepositoryFeed.vue

```diff
@@ -378,6 +396,8 @@
           url: this.newtestRepositoriesUrl,
           tests_dir: this.newtestRepositoriesDir,
           tests_prefix: this.newtestRepositoriesPrefix,
+          team_id: this.current_team.id,
+          owner_id: this.$store.state.users.self.user_id,
```

Comment:
I am not sure we really need to pass this parameter to the web server

Reply:
When is and the dependencies are removed the code breaks so I believe it is needed. My guess is that it has to do with deletion which can only be done by whoever creates the repo.

Reply:
I do agree with @maccelf, instead, we should not send `owner_id` over and manually update the `TestRepositoryCreate` payload with current user in backend. See [this related review comment](https://github.com/AlmaLinux/albs-web-server/pull/855#discussion_r1637201404) on the albs-web-server side of this PR

Reply:
Again, please remove this variable from the payload. And get owner ID on the backend side from jwt token as you did for PATCH endpoint. Because this is very not secure send owner ID via payload, someone can replace it.